app/src/test/java/com/litewallet/currency/CurrencyTests.kt

```diff
@@ -0,0 +1,102 @@
+package com.litewallet.currency
+import android.content.Context
+import android.database.sqlite.SQLiteDatabase
+import com.breadwallet.tools.sqlite.CurrencyDataSource
+import com.breadwallet.tools.util.LocaleHelper
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.verify
+import kotlinx.coroutines.delay
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNotNull
+import org.junit.Test
+
+class CurrencyTests {
+
+    @Test
+    fun `invoke CurrencyDataSource instance and getAllCurrencies, should return the correct number of currencies`() {
```

Comment:
@andhikayuana  these are my attempts at the tests and I need your comments to fix them.  They do not run correctly